README.md

```diff
@@ -619,6 +619,12 @@ Mike Grabowski
 Topics: React Native, JavaScript, Tooling  
 https://twitter.com/grabbou
 
+<img src="https://pbs.twimg.com/profile_images/813866423214870528/ct8REef9_400x400.jpg" height="70px" width="auto" align="left" alt="" />
+
+Mike Pierzchala  
```

Comment:
nit: `Michal Pierzchala` (or `Michał Pierzchała` if you fancy non-latin chars, I'm OK with both)

I'm also available in Krakow :D but nevermind